build.gradle.kts

```diff
@@ -9,10 +10,14 @@ repositories {
     mavenCentral()
 }
 
-dependencies {
-    testCompile("junit", "junit", "4.12")
+subprojects{
+    apply(plugin="java")
+    dependencies{
+        testImplementation("org.junit.jupiter:junit-jupiter:"+junitVersion)
```

Comment:
can be written as `testImplementation("org.junit.jupiter:junit-jupiter:$junitVersion")